### DISCUSSION TITLE
🌟 Looking for Collaborators at co-log organization

### DISCUSSION THREAD

**Discussion by vrom911** [kowainik/co-log#255] [General]
Hello amazing developers!

We are thrilled to announce that the [co-log GitHub Organization](https://github.com/orgs/co-log) is actively seeking collaborators to join our mission of building exceptional software libraries. 🎉

If you share our enthusiasm for crafting robust and reliable software solutions, we invite you to join us and become a collaborator at the co-log. 

Here's what you would be able to be part of:

🔧 Maintain and Improve Libraries: Help us maintain our existing libraries by fixing bugs, introducing improvements, and participating in general clean-up activities. Your expertise and contributions will ensure the continued success and stability of our projects.

🚀 Create New Releases: Be involved in the release process, ensuring that our libraries are packaged and delivered to developers worldwide. Help us create well-documented and user-friendly releases that empower others to leverage our software effectively.

💡 Contribute to Project Direction: Join the decision-making process and have a say in the direction of our projects. Collaborate with other team members to shape the roadmap, prioritize features, and define the future of our software libraries.

🔍 Participate in Code Review: Engage in meaningful code reviews, providing constructive feedback and ensuring the quality and maintainability of our codebase. Your input will help elevate the overall standard of our projects and promote best practices.

🌱 Learn and Grow Together: Collaborate with talented developers, expand your skill set, and deepen your knowledge. Engage in technical discussions, share insights, and benefit from the mentorship and guidance of experienced contributors.

If you are interested in doing so and want to become part of the organisation, feel free to contact me at vrom911@gmail.com and I would give you access to the organisation and discuss further arrangements as well.

Looking forward to work with you all! 😍

**Comment by alaendle** [comment]
Count me in 😄 I don't want to promise too much, but as before, I'm very interested in the ongoing maintenance of the libraries.

**Reply by vrom911** [reply]
Thanks so much ❤️ This is extremely valuable!
 I send the invitation :) 
 Let me know your Hackage name if you want to participate in release processes as well!

**Reply by alaendle** [reply]
I took this as an opportunity to create a hackage account - same username - alaendle.

**Reply by vrom911** [reply]
Thanks! Added you to `co-log`, `co-log-core` and `co-log-polysemi` 🎉 Thanks so much for the help 🙂

**Reply by vrom911** [reply]
We can discuss the release process. I will write up some notes on that this week :)

**Reply by alaendle** [reply]
Before thinking about release, we should also discuss how to incorporate code changes (think of review; build; etc.). E.g. on https://github.com/co-log/co-log-polysemy/pull/16 im unable to promote the PR (e.g. self-review 😉 or implicitly start CI - or even merge). My view: 
![grafik](https://github.com/co-log/co-log/assets/969523/6dbbbf57-dd2b-4551-b02e-7b202c3e5f51)
I've just taken this as an example, because I think the change is rather uncontroversial and relativ old. Nobody, especially not me, should wait that long for feedback 😉 .

**Reply by alaendle** [reply]
🔔

**Comment by develop7** [comment]
I'd love to offer at least a couple of hours per week of my time to help this project evolve and gain the traction it deserves.
